db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/user_add_vm_template_finished_failure.yaml

```diff
@@ -7,6 +7,4 @@ object:
     name: USER_ADD_VM_TEMPLATE_FINISHED_FAILURE
     inherits: 
     description: 
-  fields:
-  - rel4:
-      value: "/System/event_handlers/event_action_policy?target=src_vm&policy_event=vm_discover&param="
+  fields: []
```

Comment:
OK. Should we then also remove it from https://github.com/ManageIQ/manageiq/blob/master/config/event_handling.tmpl.yml#L54?